frontend/src/lib/WorkflowParser.tsx

```diff
@@ -247,4 +262,14 @@ export default class WorkflowParser {
       return '';
     }
   }
+
+  private static _placeholderNodeIcon(): JSX.Element {
```

Comment:
Consider moving this into a component under atoms?

Reply:
Not opposed to that, but it's not going to be reused anywhere at this point, do you think it makes sense to pull it out?

Reply:
I've a slight preference to that from an organizational point of view, otherwise this is a nit, up to you.

Reply:
Sounds good. Added `IconWithTooltip` which I think will be useful elsewhere